packages/carbon-web-components/src/components/tile/tile-story.ts

```diff
@@ -25,38 +22,22 @@ const colorSchemes = {
 };
 
 export const Default = (args) => {
-  const { colorScheme } = args?.['bx-tile'] ?? {};
+  const { colorScheme } = args?.['cds-tile'] ?? {};
```

Comment:
I've seen `${prefix}` being used in other components and it's probably the right move.
https://github.com/emyarod/carbon-for-ibm-dotcom/pull/11/files#diff-d1a8a41b1a39b5dc081b26b357a808b51f973ba5ce2ce6b6ed89413eda63b74eR17

@emyarod you've only been using `cds` in story knobs, yeah?

Reply:
yep I've been changing to `cds` in each story